velox/dwio/parquet/tests/writer/ParquetWriterTestBase.cpp

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/dwio/parquet/tests/writer/ParquetWriterTestBase.h"
+
+namespace facebook::velox::parquet {
+
+std::vector<RowVectorPtr> ParquetWriterTestBase::createBatches(
+    const RowTypePtr& rowType,
+    uint64_t numBatches,
+    uint64_t vectorSize) {
+  std::vector<RowVectorPtr> batches;
+  batches.reserve(numBatches);
+  VectorFuzzer fuzzer({.vectorSize = vectorSize}, leafPool_.get());
```

Comment:
By default, fuzzer doesn't generate nulls. Is this the desired behavior here?

Reply:
We just need some data to write here to check close/abort here.

Reply:
@duanmeng we need nulls for other tests.
@mbasmanova How do we currently generate nulls along with the Fuzzer for other tests?

Reply:
@majetideepak Fuzzer allows to specify a fraction of nulls:

```
class VectorFuzzer {
 public:
  struct Options {
    size_t vectorSize{100};

    /// Chance of generating a null value in the output vector (`nullRatio` is a
    /// double between 0 and 1).
    double nullRatio{0};
```